types/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "undici-types",
+  "version": "5.24.0",
+  "description": "A stand-alone types package for Undici",
+  "homepage": "https://undici.nodejs.org",
+  "bugs": {
+    "url": "https://github.com/nodejs/undici/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nodejs/undici.git"
+  },
+  "license": "MIT",
+  "types": "index.d.ts",
+  "author": "Matteo Collina <hello@matteocollina.com>",
```

Comment:
Maybe list all collaborators here and keep them in sync? This has gone a way forward. In fact I'm not the first collaborator anymore (bypassed by @KhafraDev and @ronag): https://github.com/nodejs/undici/graphs/contributors